site/profile/manifests/puppet/cd4pe_buildserver.pp

```diff
@@ -0,0 +1,7 @@
+class profile::puppet::cd4pe_buildserver {
+  case $::kernel {
```

Comment:
Style nit: style guide recommends using `$facts` where possible, to make it clearer to readers that this is a fact variable. Equivalent would be `case $facts['kernel'] {`.

It's a style thing so not too worried about it. Pointing it out in case conformance to style floats your boat and you want to go on a fact-hunting mission :)

Reply:
fixed